src/components/Measures/GeneralMeasures.js

```diff
@@ -89,6 +89,10 @@ export const GeneralMeasures = ({ riskLevel }) => {
         jedoch, dass pro Bundesland und Landkreis diese teilweise abweichen
         können.
       </p>
+      <p>
+        Alle bis zum 31. Januar befristeten Maßnahmen sind vorläufig bis zum 14.
```

Comment:
I thiiink, this component is generally unused right now. Does not hurt to add this there but I think it won't be shown.

Reply:
Recognized it after changing, could'nt know.